tests/testthat/test_smoothingFilters-functions.R

```diff
@@ -3,11 +3,15 @@ context("smoothingFilters")
 test_that(".movingAverage", {
   values <- c(rep(3, 3), 4:7, rep(8, 3))
   expect_equal(MALDIquant:::.movingAverage(1:10, halfWindowSize=2), values)
+  expect_equal(MALDIquant:::.movingAverage(1:10, halfWindowSize=2, 
+    weighted=TRUE), values)
 
   values <- c(rep(4, 4), 5:6, rep(7, 4))
   expect_equal(MALDIquant:::.movingAverage(1:10, 3), values)
+  expect_equal(MALDIquant:::.movingAverage(1:10, 3, weighted=TRUE), values)
```

Comment:
Could you please add an unit test where the default and the weighted version differ?

Reply:
Good idea. Done.